packages/ui/src/components/molecules/TempListItem.tsx

```diff
@@ -0,0 +1,199 @@
+import React, { FC, ReactElement } from 'react';
+import styled from 'styled-components';
+
+import {
+  CheckBox,
+  Image,
+  RadioButton,
+  Tag,
+  Typography,
+  TypographyColor,
+} from '../atoms';
+
+export interface TempListItemProps {
+  leftImageSrc?: string;
+  rightImageSrc?: string;
+  rightText?: string;
+  isHovered?: boolean;
+  tag?: string;
+  text: string;
+  displayRadioButton?: boolean;
+  radioButtonValue?: string;
+  restrictedItem?: boolean;
+  rightTextColor?: TypographyColor;
+  textVariant?:
+    | 'h1'
+    | 'h2'
+    | 'h3'
+    | 'h4'
+    | 'h5'
+    | 'h6'
+    | 'p'
+    | 'span'
+    | 'fineprint';
+  rightTextVariant?:
+    | 'h1'
+    | 'h2'
+    | 'h3'
+    | 'h4'
+    | 'h5'
+    | 'h6'
+    | 'p'
+    | 'span'
+    | 'fineprint';
+  checkBox?: boolean;
+  onCheckBoxChange?: (isChecked: boolean) => void;
+  id?: string;
+  forceUncheck?: boolean;
+  onClick?: () => void;
+  selectedItem?: TempListItemProps | null;
```

Comment:
We are only using `id` from this property, so I don't think whole object is necessary

Reply:
changed in here and also in DropDown.tsx as prop type would completely change